### PR TITLE
Not able to build SDK on local

### PR DESCRIPTION
**Pull request by MartinM85**
Fix for #2452 
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoftgraph/msgraph-sdk-dotnet/pull/2553)

### CODE REVIEW

File: src/Microsoft.Graph/Microsoft.Graph.csproj [hunk 1]
```diff
@@ -36,6 +36,7 @@ https://github.com/microsoftgraph/msgraph-sdk-dotnet/blob/dev/CHANGELOG.md
     <Deterministic>true</Deterministic>
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <NoWarn>$(NoWarn);NU5048;NETSDK1138</NoWarn>
+    <Version>$(VersionPrefix)</Version>
```
Comment (baywet) [inline src/Microsoft.Graph/Microsoft.Graph.csproj:39]: could we add a condition: if the suffix is empty?

**Review by baywet — CHANGES_REQUESTED** [review]

**Review by baywet — APPROVED** [review]

**Comment by baywet** [comment]
@andrueastman for final review and merge

**Comment by andrueastman** [comment]
We just need to validate the scripts below are not broken by this change.
 
- https://github.com/microsoftgraph/msgraph-sdk-dotnet/blob/dev/scripts/IncrementMinorVersion.ps1
- https://github.com/microsoftgraph/msgraph-sdk-dotnet/blob/dev/scripts/ValidateUpdatedNugetVersion.ps1

**Comment by MartinM85** [comment]
Executed both scripts manually on local...seems like no issues

image

**Review by baywet — APPROVED** [review]
Thanks for checking!

**Comment by baywet** [comment]
@MartinM85 can you replicate those changes on github.com/microsoftgraph/msgraph-beta-sdk-dotnet please?

**Comment by MartinM85** [comment]
@baywet [Microsoft.Graph.Beta.csproj](https://github.com/microsoftgraph/msgraph-beta-sdk-dotnet/blob/0944ee89ddea63d783b93fdff56b42e1a5f1c1aa/src/Microsoft.Graph/Microsoft.Graph.Beta.csproj#L35) has `VersionSuffix` set to **preview**. NuGet restore works fine from Visual Studio for https://github.com/microsoftgraph/msgraph-beta-sdk-dotnet.

**Comment by baywet** [comment]
Thanks for double checking, no work required then!

**Pull request merged**